src/downshift.js

```diff
@@ -67,7 +68,12 @@ class Downshift extends Component {
     defaultIsOpen: false,
     getA11yStatusMessage,
     id: generateId('downshift'),
-    itemToString: i => (i == null ? '' : String(i)),
+    itemToString: i => {
+      if (isPlainObject(i)) {
+        console.warn('you passed an Object as the value of `selectedItem`. Please refer to `itemToString` API documentation on Downshift\'s repository.')
```

Comment:
I think prettier should have reformatted this automatically when you committed. Did you `npm install`?